VTEX - Orders API.json

```diff
@@ -578,7 +578,7 @@
                                                             },
                                                             "issuanceDate": {
                                                                 "type": "string",
-                                                                "description": "Issuance date."
+                                                                "description": "The invoice date is when the order was [invoiced](https://help.vtex.com/en/tracks/orders--2xkTisx4SXOWXQel8Jg8sa/2WgQrlHTyVo4hLjhUs1LMT) in the VTEX platform, and the `IssuanceDate` is the date when the merchant registers the order's invoice in the store's system. The `IssuanceDate` can be different from the invoice date, because the merchant can register the invoice days or even months after order invoicing."
```

Comment:
```suggestion
                                                                "description": "The invoice date is when the order was [invoiced](https://help.vtex.com/en/tracks/orders--2xkTisx4SXOWXQel8Jg8sa/2WgQrlHTyVo4hLjhUs1LMT), and the `IssuanceDate` is the date when the merchant registers the order's invoice in the VTEX system. The `IssuanceDate` can be different from the invoice date, because the merchant can register the invoice days or even months after order invoicing."
```